grokproject/templates.py

```diff
@@ -1,7 +1,16 @@
+from __future__ import print_function
 import sys
 import os
-import urllib2
-import urlparse
+try:
+    import urllib2
+    import urlparse
+    from urlparse import urljoin
+    from urllib2 import HTTPError, urlopen
+except ImportError:
+    import urllib as urllib2
+    from urllib.parse import urlparse, urljoin
+    from urllib.error import HTTPError
+    from urllib.request import urlopen
```

Comment:
Might be better handled through `six.moves`?

Reply:
Indeed.  Thanks for the pointers.  Done.